src/psyclone/dynamo0p3_builtins.py

```diff
@@ -568,19 +595,19 @@ def gen_code(self, parent):
 
         '''
         # We subtract each element of f1 from the corresponding element of f2
-        # and store the result back in f1.
+        # and store the result back in f1 (real-valued fields).
         field_name1 = self.array_ref(self._arguments.args[0].proxy_name)
         field_name2 = self.array_ref(self._arguments.args[1].proxy_name)
         parent.add(AssignGen(parent, lhs=field_name1,
                              rhs=field_name1 + " - " + field_name2))
 
 
 class DynAXMinusYKern(DynBuiltIn):
-    ''' Z = a.X - Y where 'a' is a scalar and 'Z', 'X' and
-    'Y' are fields '''
+    ''' Z = a.X - Y where 'a' is a real scalar and 'Z', 'X' and
+    'Y' are real-valued fields '''
```

Comment:
Full stop and blank line inside docstring please.

Reply:
As above, done for all built-ins.